io.github.tomluchowski.OpenDungeonsPlus.yml

```diff
@@ -0,0 +1,151 @@
+app-id: io.github.tomluchowski.OpenDungeonsPlus
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+command: /app/games/opendungeons-plus
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --share=network
+  - --device=dri
+  - --env=LD_LIBRARY_PATH=/app/lib64:/app/lib
+  - --socket=pulseaudio    # Allow access to PulseAudio  
+
+modules:
+  # Boost module (for various utilities)
+  - name: boost
+    buildsystem: simple
+    build-commands:
+      - ./bootstrap.sh --prefix=/app
+      - ./b2 install
+    sources:
+      - type: archive
+        url: https://boostorg.jfrog.io/artifactory/main/release/1.82.0/source/boost_1_82_0.tar.gz
+        sha256: 66a469b6e608a51f8347236f4912e27dc5c60c60d7d53ae9bfe4683316c6f04c
+
+
+
+  # OIS module (for input system)
+  - name: OIS
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+    sources:
+      - type: archive
+        url: https://github.com/wgois/OIS/archive/refs/tags/v1.5.tar.gz
+        sha256: aa3e6b840b1149bb06835e04365957f4659c08b3e3095a48b289050f1637a174
+
+
+
+  # PugiXML module (added to fix the missing pugixml.hpp error)
+  - name: pugixml
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+    sources:
+      - type: archive
+        url: https://github.com/zeux/pugixml/archive/refs/tags/v1.13.tar.gz
+        sha256: 5c5ad5d7caeb791420408042a7d88c2c6180781bf218feca259fd9d840a888e1  # Replace with actual checksum
+
+
+
+  # pybind11 module (to fix missing pybind11)
+  - name: pybind11
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DPYBIND11_FINDPYTHON=ON
+      - -DPYTHON_EXECUTABLE=/usr/bin/python3      
+      - -DPYTHON_INCLUDE_DIR=/usr/include/python3.10  # Specify Python include path
+      - -DPYTHON_LIBRARIES=/usr/lib/x86_64-linux-gnu/libpython3.10.so  # Specify Python library path
```

Comment:
this doesn't exist. Also how does this work on aarch64?

Reply:
I don't have access to aarch64 build, how I am supposed to know ?

Reply:
first setting a `x86_64` specific path only, either you don't need it or it will break on non `x86_64`. I'm not the developer here.

Reply:
- -DPYBIND11_FINDPYTHON=ON is enough